release.py

```diff
@@ -0,0 +1,474 @@
+'''
+Make sure you follow the repos from this page: https://app.circleci.com/projects/project-dashboard/github/reactioncommerce/
+'''
+import subprocess
+import semantic_version
+import os
+from collections import defaultdict
+from git import Repo
+import fileinput
+import time
+from shutil import copyfile
+import sys
+import requests
+
+CORE_MEMBERS = ['akarshitwal@gmail.com', 'jessica.wolvington@gmail.com']
```

Comment:
+ Mohan

Reply:
Mohan is using his Mailchimp email for the commits, so he should be fine.

Reply:
Ah, I see. Sounds good!